frontend/packages/dev-console/src/components/topology/__tests__/topology-knative-test-data.ts

```diff
@@ -176,8 +182,8 @@ const sampleKnativeConfigurations: Resource = {
 const sampleKnativeRevisions: Resource = {
   data: [
     {
-      apiVersion: 'serving.knative.dev/v1alpha1',
-      kind: 'Revision',
+      apiVersion: `${RevisionModel.apiGroup}/${RevisionModel.apiVersion}`,
+      kind: 'RevisionModel',
```

Comment:
@joshuawilson tests are failing as here kind should be `RevisionModel.kind`.

Reply:
thanks